src/main/java/edu/ucsb/cs156/courses/controllers/PersonalSchedulesController.java

```diff
@@ -81,7 +81,7 @@ public PersonalSchedule getScheduleById_admin(
     @ApiOperation(value = "Create a new personal schedule")
     @PreAuthorize("hasRole('ROLE_USER')")
     @PostMapping("/post")
-    public PersonalSchedule postSchedule(
+    public Object postSchedule(
```

Comment:
Just curious, why did the signature need to change? It seems to compile fine for me with PersonalSchedule return type.

Reply:
I'm wondering that too.

Reply:
It would probably be better to change it back unless you have a specific reason that it must change to Object.